src/docs/metrics/nft-collection-price/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: NFT Collection Price
+author: Filip
+date: 2022-06-17
+description: Daily minimum, maximum, average price in eth and usd, and daily trade count for NFT collections and tokens
```

Comment:
Make another academy for daily trade count

Reply:
Fixed